nerdlets/container-explorer/container-explorer.js

```diff
@@ -232,7 +232,6 @@ export default class ContainerExplorer extends React.Component {
                 showRelatedApps
                 onClose={() => this.setState({ containerId: null })}
               />
-              } />
```

Comment:
Was this an intentional change? Seems kinda random

Reply:
It's possible it was my prettier linter, but let me check that line of code.

Reply:
I think it just removed unnecessary code in this case.